notebooks/tensorflow/getting-started-keras.ipynb

```diff
@@ -245,7 +245,7 @@
         "\n",
         "2. [Make sure that billing is enabled for your project.](https://cloud.google.com/billing/docs/how-to/modify-project)\n",
         "\n",
-        "3. [Enable the AI Platform and Compute Engine APIs.](https://console.cloud.google.com/flows/enableapi?apiid=ml.googleapis.com,compute_component)\n",
+        "3. [Enable the AI Platform ("Cloud Machine Learning Engine") and Compute Engine APIs.](https://console.cloud.google.com/flows/enableapi?apiid=ml.googleapis.com,compute_component)\n",
```

Comment:
The use of un-escaped quotation marks here resulted in invalid JSON.

Reply:
Thanks! Addressed in #401